whisper.go

```diff
@@ -878,6 +878,26 @@ func (whisper *Whisper) UpdateMany(points []*TimeSeriesPoint) (err error) {
 	return whisper.UpdateManyForArchive(points, -1)
 }
 
+/*
+  Returns updated amount of discard points since opening whisper file
+*/
+func (whisper *Whisper) GetDiscardedPointsSinceOpen() uint32 {
+	var discardedPointsNow uint32
+	discardedPointsNow = 0
```

Comment:
just in case, this is not really needed because go has [zero value at init](https://go.dev/tour/basics/12).

Reply:
Ah, forgot about that. Would better remove it then.